README.md

```diff
@@ -234,3 +235,17 @@ Please read the [docs](https://github.com/IBM/nodejs-idb-pconnector/blob/master/
 Please read the [contribution guidelines](https://github.com/IBM/nodejs-idb-pconnector/blob/master/CONTRIBUTING.md).
 
 
+# **Release**
+
+To generate a release use one of the npm scripts:
+
+- patch release `npm run release`
+- for other release types run `npm run release-interactive` which will prompt
+  you for the release type.
+
+These scripts will bump the version number, make a release commit, and tag the release.
+
+Once the tag is created, create a github release manually using the appropriate tag.
```

Comment:
I think for idb-pconnector, we'd be fine with having release-it make the GH release since it doesn't need to build and attach any native objects. If a user has GITHUB_TOKEN set it would do this automatically, otherwise it would spit out a URL with all the fields pre-populated which would save time too: https://github.com/release-it/release-it/blob/HEAD/docs/github-releases.md#manual

Alternatively, we could run a GH Action to do it on tag creation instead. Update the GH Action to trigger on tag creation of vX.Y.Z, create a GH release, then publish to npm.

What do you think?

Reply:
I like having release it generate the github release. Especially since it has a fallback to pre-populate manual github release when the user does not have a GITHUB_TOKEN set